packages/nouns-contracts/src/merkleAirdrop.ts

```diff
@@ -0,0 +1,14 @@
+import { StandardMerkleTree } from "@openzeppelin/merkle-tree";
+
+export const generateMerkleTree = (values: any[][]): StandardMerkleTree<any> => {
+    return StandardMerkleTree.of(values, ["address"])
+};
+
+export const generateMerkleProof = (tree: StandardMerkleTree<any>, address: string): string[] => {
+  for (const [i, v] of tree.entries()) {
+    if (v[0] == address) {
```

Comment:
we should check the lowercase of the address. In case the upper level passes down a lowercase wallet address but the v[0] has uppercase and lowercase of the wallet address.

Reply:
fixed

Reply:
I double checked generateMerkleTree to verify that string casing doesn't matter in encoding.